crates/katana/core/src/sequencer.rs

```diff
@@ -423,7 +423,7 @@ impl Sequencer for KatanaSequencer {
             .ok_or(SequencerError::BlockNotFound(to_block))?;
 
         let mut events = Vec::new();
-        for i in from_block.0..to_block.0 {
+        for i in from_block.0..to_block.0 + 1 {
```

Comment:
```suggestion
        for i in from_block.0..=to_block.0 {
```

Reply:
Wow, didn't know that was possible. That's even better. :) Thanks @kariy.